Source/Localization/en.lproj/Generated.strings

```diff
@@ -18,6 +18,7 @@
 "Text.future" = "In the future";
 "Text.daysAgo" = "{{ days }} days ago";
 "Text.atDayTime" = "{{ day }} at {{ time }}";
+"Text.minutesAbbreviated" = "{{ numberOfMinutes }} mins";
```

Comment:
We can add the Spanish and Polish translations, I can obviously give a Spanish translation. But I think that Sebastian really like to translate texts. Maybe we can create a channel in slack where we can past texts in English and ask the rest to give translations.

Reply:
Filed #408